src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
I disagree. The given implementation didn't included any of the necessary steps to actually load plugins and only provided the needed part to test the plugin interface.

As said, I would not share those objects (global) as I don't see the need nor any value at the moment.